### DISCUSSION TITLE
How do I show the 'citation' block at the end of my book chapters?

### DISCUSSION THREAD

**Discussion by chrisvoncsefalvay** [quarto-dev/quarto-cli#7646] [Q&A] [answered]
### Description

I have a book with a properly specified citation in the _quarto.yml file. However, I can't get it to show the BibTeX citation & attribution block at the end of each chapter. What am I doing wrong?

**Comment by mcanouil** [comment]
Note that a very small search on Quarto website in the extension listing page and/or in GitHub discussion would have given you the answer.

https://github.com/pandoc-ext/section-bibliographies

> However, I can't get it to show the BibTeX citation & attribution block at the end of each chapter. What am I doing wrong?

And if that is not the right answer for your case, you need to share something that we can work with.
You did not tell or show anything, so we can't tell you anything unfortunately.

**Comment by chrisvoncsefalvay** [comment]
No, that gives me _my_ citations. I want the block that shows the citation to my current page.

**Reply by mcanouil** [reply]
Still unclear what you are referring to.
Please provide an example!

Are you talking about https://quarto.org/docs/reference/formats/html.html#citation ?

**Reply by chrisvoncsefalvay** [reply]
No, this: https://quarto.org/docs/authoring/create-citeable-articles.html

**Reply by cscheid** [reply]
> However, I can't get it to show the BibTeX citation & attribution block at the end of each chapter

I don't think that feature exists for book chapters.

**Reply by cderv** [reply, marked as answer]
I believe this is a feature requests track here already: 

* https://github.com/quarto-dev/quarto-cli/issues/3838
